app/assets/javascripts/dialog-polyfill.js

```diff
@@ -0,0 +1,533 @@
+(function() {
```

Comment:
@tansaku I did not write it. Got it from [https://github.com/GoogleChrome/dialog-polyfill](url). It is a requirement to be able to run 'Dialogs' (redeem your code and delete your account dialog boxes) in browsers other than Chrome.

I'm not sure where it should be placed. Can you advise? Thx.